test/runtests.jl

```diff
@@ -1144,27 +1493,102 @@ const prob_pds_linmod_nonconservative_inplace = PDSProblem(linmodP!, linmodD!, [
         end
     end
 
-    # TODO: Do we want to keep the examples and test them or do we want
-    #       to switch to real docs/tutorials instead?
-    @testset "Examples" begin
-        # dummy test to make sure the testset errors if the process
-        # errors out
-        @test true
-
-        if VERSION >= v"1.10"
-            cmd = Base.julia_cmd()
-            examples_dir = abspath(joinpath(pkgdir(PositiveIntegrators), "examples"))
-            examples = ["01_example_proddest.jl",
-                "02_example_mpe.jl",
-                "03_example_mprk22.jl",
-                "04_example_problemlibrary.jl"]
-
-            @testset "Example $ex" for ex in examples
-                @info "Testing examples" ex
-                example = joinpath(examples_dir, ex)
-                @test isfile(example)
-                @time run(`$cmd --project=$(examples_dir) $(example)`)
+    # Here we check that the implemented schemes can solve the predefined PDS
+    # (at least for specific parameters)
+    @testset "PDS problem library (adaptive schemes)" begin
+        algs = (MPRK22(0.5), MPRK22(1.0), MPRK43I(1.0, 0.5), MPRK43I(0.5, 0.75),
+                MPRK43II(2.0 / 3.0), MPRK43II(0.5), SSPMPRK22(0.5, 1.0))
+        probs = (prob_pds_linmod, prob_pds_linmod_inplace, prob_pds_nonlinmod,
+                 prob_pds_robertson, prob_pds_bertolazzi, prob_pds_brusselator,
+                 prob_pds_npzd,
+                 prob_pds_sir, prob_pds_stratreac)
+        @testset "$alg" for alg in algs
+            @testset "$i" for (i, prob) in enumerate(probs)
+                if prob == prob_pds_stratreac && alg == SSPMPRK22(0.5, 1.0)
+                    #TODO: SSPMPRK22(0.5, 1.0) is unstable for prob_pds_stratreac. 
+                    #Need to figure out if this is a problem of the algorithm or not.
+                    break
+                elseif prob == prob_pds_stratreac && alg == MPRK43I(0.5, 0.75)
+                    # Not successful on Julia 1.9
+                    break
+                end
+                sol = solve(prob, alg)
+                @test Int(sol.retcode) == 1
+            end
+        end
+    end
+
+    # Here we check that the implemented schemes can solve the predefined PDS.
+    @testset "PDS problem library (non-adaptive schemes)" begin
+        algs = (MPE(), SSPMPRK43())
+        #prob_pds_robertson not included
```

Comment:
```suggestion
        #TODO: prob_pds_robertson not included since it needs an adaptive step size
```

Reply:
Sure 👍  I just wanted to suggest to make this a proper TODO note to make it easier to find later 🙂